docs/diagrams/DoneCryptoSequenceDiagram.puml

```diff
@@ -0,0 +1,116 @@
+@startuml
+'https://plantuml.com/class-diagram
+!include Styles.puml
+
+box console COLOR_BRIGHT_GREEN
+participant ":InputParser" as InputParser COLOR_CONSOLE
+end box
+
+box command COLOR_YELLOW
+participant ":DoneCommand" as DoneCommand COLOR_COMMANDS
+end box
+
+box model COLOR_GREY
+participant ":InstrumentManager" as InstrumentManager COLOR_MODEL
+participant ":Instrument" as Instrument COLOR_MODEL
+end box
+
+box ui COLOR_LIGHT_BLUE
+participant ":TextUi" as TextUi COLOR_UI
+end box
+
+
+[-> InputParser : filterByCommandType({"done"}, instruments)
+activate InputParser
+
+InputParser-> DoneCommand : new DoneCommand()
+activate DoneCommand
+
+InputParser ->InputParser: getDoneInstrumentCommand(commandComponents[], instruments)
+activate InputParser
+
+InputParser ->InputParser: getIndex(commandComponents[])
+activate InputParser
+InputParser --> InputParser
+deactivate InputParser
+
+InputParser ->DoneCommand: setIndex(index)
+activate DoneCommand
+
+activate DoneCommand
+DoneCommand --> DoneCommand
+deactivate DoneCommand
+
+deactivate DoneCommand
+DoneCommand--> InputParser
+deactivate DoneCommand
+
+InputParser -->InputParser: return doneCommand
+
+deactivate InputParser
+[<--InputParser: return doneCommand
+deactivate InputParser
+
+[->DoneCommand:execute()
+activate DoneCommand
+
+
+DoneCommand-> InstrumentManager: getInstrument(index)
+activate InstrumentManager
+
+activate Instrument
```

Comment:
Can u check the activate order, the activation bar extends outside the point when get(Index) starts. the activation bar is suppose to start when get(index) starts.
Also no need the get(Index), its a java arraylist method, so its not exactly correct that get method is an instrument method